index.html

```diff
@@ -114,24 +116,26 @@ <h3 class="h1 text-text-bold">Tonic</h3>
             width="295"
             height="220"
           />
-          <h3 class="h1 text-text-bold">Multi-Post Stories</h3>
-          <ul class="captions">
-            <li class="text-text-bold">CANOPY</li>
-            <li><span class="counter"></span></li>
-            <li class="text-medium text-light-grey">Back End Dev</li>
-            <li><span class="counter"></span></li>
-            <li class="text-medium text-light-grey">2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="project-language">
-            <li class="text-medium text-dark-purple">HTML</li>
-            <li class="text-medium text-dark-purple">CSS</li>
-            <li class="text-medium text-dark-purple">JS</li>
-          </ul>
-          <button class="button-outlined text-medium">See project</button>
+          <div class="content">
+            <h3 class="h1 text-bold">Multi-Post Stories</h3>

```

Comment:
- [ ]  Nice work working on the desktop version 😃 , it is like wow! but... see this below, the card disposition should be reversed, if the first project has the image on the left and the text on the right, then the following one should have the text on the left and the image on the right, and so on. Kindly make sure you fixed it to replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.) 👍.

### Figma 

![Capture](https://user-images.githubusercontent.com/57408419/228794816-4cd30221-5bd7-41df-a1dc-217856c0a41d.JPG)

### Your project 

![Capture](https://user-images.githubusercontent.com/57408419/228795797-40a050d7-c057-422a-85f6-ff6faa3b42bb.JPG)


- [ ] Also make sure you implemented the `hover` action on the project 😃. As you can check  the [Figma templet](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=8aiVXoAZHHn4TTdQ-0) , there is some `box-shadow` and `boder` when we hover the section 👍 




![Capture](https://user-images.githubusercontent.com/57408419/228796904-3d8f95a7-1032-4ca8-8948-16403fefe3e5.JPG)

Reply:
alright, consider it done